RELEASE.md

```diff
@@ -1,3 +1,45 @@
+# Release 1.12.0
+
+## Major Features and Improvements
+* Keras models can now be directly exported to the SavedModel format(`tf.contrib.saved_model.save_keras_model()`) and used with Tensorflow Serving.
+* Keras model now supports evaluating with a `tf.data.Dataset`.
```

Comment:
For consistency with the first bullet this should be `Keras models now support ...`

Reply:
Done.